api/database/boltdb/db.go

```diff
@@ -70,12 +107,12 @@ func (connection *DbConnection) BackupTo(w io.Writer) error {
 }
 
 func (connection *DbConnection) ExportRaw(filename string) error {
-	databasePath := path.Join(connection.Path, DatabaseFileName)
+	databasePath := path.Join(connection.Path, connection.GetDatabaseFilename())
 	if _, err := os.Stat(databasePath); err != nil {
 		return fmt.Errorf("stat on %s failed: %s", databasePath, err)
 	}
 
-	b, err := exportJson(databasePath)
+	b, err := exportJson(databasePath, connection.getEncryptionKey())
```

Comment:
this is debug code, and not a (currently) desired feature - sorry, I wonder if we should put this behind a "debug" feature flag @hookenz ?

Reply:
@SvenDowideit do we have a debug feature flag?